src-frontend/app/components/choice-list.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    targetObject: Em.computed.alias('parentView'),
```

Comment:
Ah I copied this code from an Ember doc and didn't notice. It's shorthand for `Ember`, but I agree it should be `Ember` for consistency.